boa/src/environment/lexical_environment.rs

```diff
@@ -4,164 +4,6 @@
 //!
 //! The following operations are used to operate upon lexical environments
 //! This is the entrypoint to lexical environments.
```

Comment:
It seems that most of this file has been deleted, but tests kept. Would it make sense to just have a tests sub-module?

Reply:
Yeah, I'm not 100% sure where the code for the compile time and the runtime environments should be located now. I think it makes some sense to have all of that code close  to each other for documentation and readability. Either I will move that into the now empty environments module or into a separate file next to the realm and move the environment tests there too.

Reply:
I moved all of the environment related code to a new `environments` module and removed the old one. I think it works pretty good that way